packages/valory/skills/market_creation_manager_abci/behaviours.py

```diff
@@ -290,19 +290,22 @@ def _get_llm_response(self) -> Generator[None, None, Optional[dict]]:
         self.context.logger.info(f"Got LLM response: {result}")
         data = json.loads(result)
         valid_responses = []
-        minimum_resolution_date = datetime.datetime.fromtimestamp(
+        # Opening date for realitio oracle contract and closing date
+        # for answering question on omen market
+        minimum_opening_date = datetime.datetime.fromtimestamp(
             self.context.state.round_sequence.last_round_transition_timestamp.timestamp()
-            + (3600 * 24)
+            + (_ONE_DAY * self.params.minimum_market_time)
         )
         for q in data:
             try:
+                # Date of the outcome
                 resolution_date = parse_date_timestring(q["resolution_date"])
                 if resolution_date is None:
                     self.context.logger.error(
                         "Cannot parse datestring " + q["resolution_date"]
                     )
                     continue
-                if resolution_date < minimum_resolution_date:
+                if resolution_date < minimum_opening_date:
```

Comment:
Is this OK??

resolution_date = Date that the event happens ("occurring date")
minimum_opening_date= Opening date on Realitio (i.e., the event should be known when opening the question on REalitio)

Therefore seems the check is not OK,  no??

Reply:
Another question, what is the date the Omen market is opened? The condition should ideally be:

omen_open_date < resolution_date (ocurring date) < realitio_open_date (= omen_closing_date)

Reply:
1. The check is correct. Yes, the `resolution_date` refers to the date when the event will be occurring. Yes, `minimum_opening_date ` refers to the opening date on omen, but it's not something that is statically defined. We calculate this value based on the current date and the [minimum_market_time](https://github.com/valory-xyz/market-creator/pull/22/files/7a2e1bcf4fdeadbc900085e17c1a73a7c638ccd2#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R48) parameter. What this means is the minimum opening date needs to be at least `current_date + minimum_market_time` or greater. 

3. The same date when the tx gets executed